pyemma/msm/tests/test_oom_msm.py

```diff
@@ -0,0 +1,811 @@
+
```

Comment:
This is a great test. Anyway, just to make sure there are no surprises, I think it would be neat to extend the test

``https://github.com/markovmodel/PyEMMA/blob/devel/pyemma/msm/tests/test_msm.py``

class ``TestMSMDoubleWell`` also by an OOM-reweighted MSM. Essentially all functions should work too. This is just to make sure all functions are meaningfully callable. 

Shouldn't take much extra work to implement. Ideally we need to write only one test (plus a little bit) per MSM estimator.